crates/sncast/tests/data/scripts/script_manual_tests/src/test_declare_after_invoke.cairo

```diff
@@ -0,0 +1,45 @@
+use sncast_std::{
+    declare, deploy, invoke, call, DeclareResult, DeployResult, InvokeResult, CallResult, get_nonce,
+};
+use debug::PrintTrait;
+use starknet::{ContractAddress};
+fn main() {
+    let max_fee = 99999999999999999;
+
+    let contract_address: ContractAddress =
+        0x18abd0648d901555975edde1330cffcf37c1da1edbf20b889ba1101e70a6c22
```

Comment:
if possible, let's avoid hardcoding addresses in scripts, as they might change after cairo update and this can become very troublesome to adjust

Reply:
I'm wondering if should we leave these scripts in repo? It was added to show what I did.